src/components/clipboard-card/clipboard-card.html

```diff
@@ -9,18 +9,21 @@
     {{payProDetailsData?.host}}
   </div>
   <ion-note *ngIf="payProDetailsData && !payProDetailsData.error" item-right>
-    <div class="amount">
+    <div *ngIf="!payProDetailsData?.isFiat" class="amount">
       {{payProDetailsData?.amount | satToUnit: payProDetailsData?.coin}}
     </div>
-    <span class="date">{{ remainingTimeStr }}</span>
+    <div *ngIf="payProDetailsData?.isFiat" class="amount">
+        {{payProDetailsData?.amount | formatCurrency: payProDetailsData?.coin}} {{payProDetailsData?.coin}}
+      </div>
+    <span [ngStyle]="remainingTimeStr === 'Expired' ? { color: 'red' } : {}" class="date">{{ remainingTimeStr }}</span>
```

Comment:
Expired color is red if expired. 

Expired is translated so remainingTimeStr might not always === 'Expired', can you double check this for me?